scripts/unittests_in_container.sh

```diff
@@ -26,10 +28,19 @@ echo ""
 echo "etcd:"
 echo ""
 DATASTORE_ADDRESS=$ETCD_ADDRESS go test -run TestAuthZ*  -v -timeout 1m ./state -check.v
+etcd_authz_status=$?
 DATASTORE_ADDRESS=$ETCD_ADDRESS go test -run TestEtcd*   -v -timeout 1m ./state -check.v
+etcd_state_status=$?
 echo ""
 
 echo "consul:"
 echo ""
 DATASTORE_ADDRESS=$CONSUL_ADDRESS go test -run TestConsul* -v -timeout 1m ./state -check.v
+consul_state_status=$?
 echo ""
+
+if [[ "$consul_db_status" != "0" || "$etcd_db_status" != "0" || "$etcd_authz_status" != "0" || "$etcd_state_status" != "0" || "$consul_state_status" != "0" ]]; then
```

Comment:
Rather storing the return codes in separate variable; can it be stored in array? So the code becomes much cleaner and easy to maintain as well.